src/components/home/styledHome.js

```diff
@@ -0,0 +1,50 @@
+import styled from "styled-components";
+
+export const HomeSection = styled.section`
+    display: flex;
+    flex-direction: column;
+    width: 94%;
+    padding: 74px 0 40px 0;
+    margin: auto;
+`
+
+export const Title = styled.h1`
+    width: 59.65vw;
```

Comment:
I recommend trying max-width here. It should be responsive. Right now it grows as large as the window is, up to 1200 px